### PR TITLE
Added graphs under query metrics drop down menu

### LINKED ISSUE #50: Add Line Graphs to Query Metrics and show data with category data
Task: Add existing line graphs to ‘Query Metrics’ and show plots using category data.
Test case: As a user, I go to the ‘FR’ country, go to ‘Query Metrics’, enter the ‘Music’ category, and see line graphs specifically for music

### PR DESCRIPTION
**Pull request by DylanHan909**
## Overview
Added existing line graphs under query metrics from the graphs previously used under general metrics.

## Materials
- [Issue] https://github.com/JakinChan200/CS180_Roald/issues/50

## Pictures
image

## Manual Testing Guide
Provide details of how to manually test your changes if applicable.
- Perform npm install for dependencies in both server and client folders.
- Run npm start on both server and client folders to run the program.

### CODE REVIEW

**Comment by DylanHan909** [comment]
Additional comments: scroll wheel currently in the works to deal with the issue of the x axis being barely visible, as data is stuffed all at once with the numerous videos at the moment.

**Comment by DylanHan909** [comment]
Additional comments: Duplication removal for country endpoints mentioned in #47 necessary to remove the numerous hole and unconnected data points present in the graphs.

File: client/src/pages/Country/Country.tsx [hunk 1]
```diff
@@ -167,19 +167,46 @@ export const Country: React.FC<CountryProps> = ({ country }) => {
           onResult={getResult}
           categories={categories}
         />
-
         {catResults.length > 0 ? (
```
Comment (gobrien06) [inline client/src/pages/Country/Country.tsx:170]: NAB Can remove the original catResults in a later PR

**Review by gobrien06 — APPROVED** [review]
LGTM - see the charts changing under different categories. See the issue of the dates being too many to display, like #53 , but hoping the scroll area will fix this

**Pull request closed without merging**